main.py

```
@@ -43,6 +43,51 @@
 )
 logger = logging.getLogger(__name__)
 
+from langchain_core.messages import HumanMessage, AIMessage, SystemMessage, BaseMessage
+from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder
```

Ruff (0.8.2)

`langchain_core.prompts.ChatPromptTemplate` imported but unused

Remove unused import

(F401)